server/pxf-api/src/main/java/org/greenplum/pxf/api/filter/SupportedDataTypePruner.java

```diff
@@ -0,0 +1,66 @@
+package org.greenplum.pxf.api.filter;
+
+import org.greenplum.pxf.api.io.DataType;
+import org.greenplum.pxf.api.utilities.ColumnDescriptor;
+
+import java.util.EnumSet;
+import java.util.List;
+
+/**
+ * A tree pruner that removes operator nodes for non-supported column data types.
```

Comment:
these are Greenplum data types
```suggestion
 * A tree pruner that removes operator nodes for non-supported Greenplum column data types.
```